pennylane/plugins/tests/__init__.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This subpackage provides integration tests for the devices with PennyLane's core
+functionalities. At the moment, the tests only run on devices based on the 'qubit' model.
+
+To run the tests, navigate to the parent directory of this file and run (i.e., for 'default.qubit'):
+
+>>> python3 -m pytest tests/* --device default.qubit --shots 1234 --analytic False
```

Comment:
(note: `>>>` is reserved for python code)

```suggestion
To run the tests against a particular device (i.e., for 'default.qubit'):

.. code-block:: console

    python3 -m pytest path_to_pennylane/plugins/tests --device default.qubit --shots 1234 --analytic False

The location of your PennyLane installation may differ depending on installation method and operating
system. To find the location, you can execute the following Python code:

>>> import os
>>> import pennylane as qml
>>> print(os.path.dirname(qml.__file__))
```

Reply:
(the last three lines of python could be provided as a convenience function, e.g., `pennylane.get_device_tests()` or something along those lines)